questdb-rs/src/ingress/mod.rs

```diff
@@ -1199,6 +1426,9 @@ pub struct Sender {
     handler: ProtocolHandler,
     connected: bool,
     max_buf_size: usize,
+    default_line_protocol_version: LineProtocolVersion,
+    #[cfg(feature = "ilp-over-http")]
+    supported_line_protocol_versions: Option<Vec<LineProtocolVersion>>,
```

Comment:
Probably worth commenting why it's an `Option<Vec<..>>` and not just a `Vec<..>`.

I'm assuming that's because it's `None` if no probing was done.
I'm just afraid neither of us will rememember in 6 months time.